### PR TITLE
Add a CHANGELOG

### PR DESCRIPTION
**Pull request by gigabo**
Targeting `next`.

This addresses #202.

### CODE REVIEW

**Comment by doug-wade** [comment]
How does the CHANGELOG generation work?

File: lerna.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 {
   "lerna": "2.0.0-beta.9",
-  "version": "0.2.12"
+  "version": "0.2.12",
+  "changelog": {
```
Comment (doug-wade) [inline lerna.json]: This is awesome

**Comment by gigabo** [comment]
> How does the CHANGELOG generation work?

I made a changelog generator based on @hzoo's [`lerna changelog`](https://github.com/lerna/lerna/pull/29) PR.  Working on getting it opened up.

**Comment by gigabo** [comment]
Gonna merge this in so I can point to the CHANGELOG on `next` for what's included in the `0.3.0` beta.

**Comment by gigabo** [comment]
Changelog generator's up on npm: https://www.npmjs.com/package/lerna-changelog

**Pull request merged**